scipy/spatial/transform/rotation.py

```diff
@@ -1801,36 +1813,26 @@ def match_vectors(cls, a, b, weights=None, normalized=False):
             Vector components observed in another frame B. Each row of `b`
             denotes a vector.
         weights : array_like shape (N,), optional
-            Weights describing the relative importance of the vectors in
-            `a`. If None (default), then all values in `weights` are assumed to
-            be equal.
-        normalized : bool, optional
```

Comment:
I suppose I must retain `normalized` description for now and explain that it is deprecated (rather just completely hide it)?

@rgommers is it a right policy here?

Reply:
Yes that would be the normal way of doing that.

Reply:
Now I see a problem here. We want to change the default behavior of `match_vectors` with respect to `normalized` keyword. So it's going to be a backward incompatible change at some point.

Eventually we want to remove `normalized` and never do the normalization, but now it is done by default.

Conceptually it looks like a truly backward incompatible change. Should we simply introduce it now? I don't see any other way.